EvotoApi/Areas/Management/Models/Request/CreateManaVote.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+using Common;
+using Management.Models;
+
+namespace EvotoApi.Areas.ManagementApi.Models.Request
+{
+    [DataContract]
+    public class CreateManaVote
+    {
+        [DataMember(Name = "orgId")]
+        [Required]
+        public int OrgId { get; }
```

Comment:
Added in https://github.com/evoto-tech/EvotoApi/pull/12/commits/e2af60ca0bae4aab7d4da01f303273aef5806bac.